### PR TITLE
Adding Control D mirror of the Unified blocklist

### PR DESCRIPTION
**Pull request by yegors**
Steven, thanks for entertaining the PR.

### CODE REVIEW

**Comment by StevenBlack** [comment]
Thank you for this Yegor @yegors.

Two things.

* **First**: this repo has 16 readme files, one for each variant generated with each release. The file you need to edit is [`readme_template.md`](https://github.com/StevenBlack/hosts/blob/master/readme_template.md).
* **Second**: Please put this in the [Interesting Applications section](https://github.com/StevenBlack/hosts/blob/master/readme_template.md#interesting-applications). Please take only as much space as the other items presently under this sections. That's 3-lines max.

Thanks!

File: readme.md [hunk 1]
```diff
@@ -581,6 +581,15 @@ devices under a variety of operating systems.
...
   written in Java, by Christian Martínez. Check it out!
+  
+## Anycast Mirrors
+- [Control D](https://controld.com/free-dns)
+  offers a public anycast network hosted mirror of the Unified (Adware + Malware) blocklist using the following endpoints:
+  - IPv4: `76.76.2.35` and `76.76.10.35`
+  - IPv6: `2606:1a40::35` and `2606:1a40:1::35`
+  - DNS-over-HTTPS: `https://freedns.controld.com/x-stevenblack`
+  - DNS-over-TLS: `x-stevenblack.freedns.controld.com`
+  - [Source](https://kb.controld.com/en/3rd-party-filters)
```
Comment (StevenBlack) [inline readme.md:593]: Please see my comment here: https://github.com/StevenBlack/hosts/pull/2075#issuecomment-1237549759

**Review by StevenBlack — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (yegors) [reply readme.md]: I tweaked the PR, is this OK?

**Comment by StevenBlack** [comment]
Thanks Yegor @yegors yes, this is good!

Merging.

**Pull request merged**